Task5/css/main/tour/__old-price.css

```diff
@@ -0,0 +1,15 @@
+.tour__old-priсe {
+    visibility: hidden;
+}
+
+.sale-label .tour__old-priсe {
+    font-size: 18px;
+
+    visibility: unset;
```

Comment:
As I remember, `unset` valu isn't supported in IE.

Reply:
Corrected